batcher/aligned-batcher/src/lib.rs

```diff
@@ -467,16 +467,6 @@ impl Batcher {
             return Ok(());
         }
 
-        // Nonce and max fee verification
-        let max_fee = nonced_verification_data.max_fee;
-        if max_fee < U256::from(MIN_FEE_PER_PROOF) {
```

Comment:
There is missing remove MIN_FEE_PER_PROOF variable

Reply:
@JuArce the MIN_FEE_PER_PROOF is used here as well: https://github.com/yetanotherco/aligned_layer/blob/testnet/batcher/aligned-batcher/src/lib.rs#L617 .

Reply:
I think if it is used in another place it will have the problem in some point